auslib/test/common/base.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import logging
+
+from auslib.blobs.base import createBlob
+from auslib.global_state import dbo
+from auslib.web.public.base import app
+from auslib.web.admin.base import app as admin_app
+
+
+def setUpModule():
+    # Silence SQLAlchemy-Migrate's debugging logger
+    logging.getLogger('migrate').setLevel(logging.CRITICAL)
+
+
+class CommonTestBase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Error handlers are removed in order to give us better debug messages
+        cls.error_spec = app.error_handler_spec
+        # Ripped from https://github.com/mitsuhiko/flask/blob/1f5927eee2288b4aaf508af5dc1f148aa2140d91/flask/app.py#L394
+        app.error_handler_spec = {None: {}}
+
+    @classmethod
+    def tearDownClass(cls):
+        app.error_handler_spec = cls.error_spec
+
+    def setUp(self):
+        app.config['DEBUG'] = True
+        self.public_client = app.test_client()
+
+        admin_app.config["WTF_CSRF_ENABLED"] = False
+        self.admin_client = admin_app.test_client()
```

Comment:
Indeed, my intention was test exactly like admin, to ensure that all is ok.
I'll refactor all test cases.